src/documenti/esterni/AdR/contenuti/introduzione.tex

```diff
@@ -22,11 +22,14 @@ \subsection{Scopo}
 
 \subsection{Attori}
 Il prodotto verrà utilizzato solo per scopo di sorveglianza interno, senza il bisogno di figure particolari per il suo funzionamento
-come un amministratore o un utente registrato, perciò sarà presente solo un attore all'interno del sistema: un utente generico, che avrà accesso a tutte
-le funzionalità del prodotto. 
+come un amministratore o un utente registrato, perciò saranno presenti solo due attori all'interno del sistema:
+\begin{itemize}
+  \item Un \textbf{utente}, che avrà accesso a tutte le funzionalità del prodotto.
+  \item Uno \textbf{sviluppatore}, che è generalizzato dall'utente e può, inoltre, accedere al codice del software, per eventuali modifiche.
```

Comment:
Non mi convince al 100% perchè sembra quasi che lo sviluppatore abbia dei privilegi diversi dall'utente normale, ma in realtà è solo un utente più esperto che può essere interessato alla modifica del codice. Io cambierei un po' la frase in questo senso, dimmi cosa ne pensi.